client.js

```diff
@@ -1,31 +1,47 @@
-function u8tohex (arr) {
-  var vals = [ '0', '1', '2', '3', '4', '5', '6', '7', '8', '9', 'a', 'b', 'c', 'd', 'e', 'f' ]
-  var ret = ''
-  for (var i = 0; i < arr.length; ++i) {
-    ret += vals[(arr[i] & 0xf0) / 0x10]
-    ret += vals[(arr[i] & 0x0f)]
+function MonetizerClient (opts) {
+  var domain = new URL(window.location).origin
+  this.url = domain
+  if (opts && opts.url) {
+    this.url = opts.url
   }
-  return ret
-}
+  this.cookieName = (opts && opts.cookieName) || '__monetizer'
+  this.receiverUrl = this.url + '/__monetizer/:id'
+
+  const COOKIE_REGEX = new RegExp(this.cookieName + '=(.*?)(;|$)')
```

Comment:
this constant can go at the top of the file so it's just declared once

Reply:
Sorry I don't quite follow - is it not already declared once?